pkg/option/config.go

```diff
@@ -340,6 +340,9 @@ const (
 	// LogOpt sets log driver options for cilium
 	LogOpt = "log-opt"
 
+	// EnableRemoteNodeSnat SNAT IPv4 packets from endpoints leaving the host destined to a remote node node in BPF routing mode. This option requires to set enable-bpf-masquerade to true.
+	EnableRemoteNodeSnat = "enable-remote-node-snat"
```

Comment:
```suggestion
	// EnableRemoteNodeMasquerade Masquerade packets from endpoints leaving the host destined to a remote node node in BPF routing mode. This option requires to set enable-bpf-masquerade to true.
	EnableRemoteNodeMasquerade = "enable-remote-node-masquerade"
```

Sorry for being pedantic, but I'd really like us to use "masquerading" in this context, so that it matches the other option names.